Tests/test_MarkovModel.py

```diff
@@ -281,54 +283,59 @@ def test_save_and_load(self):
         output_p_transition = array(
-            [[5.16919807e-001, 3.65825814e-033, 4.83080193e-001, 9.23220689e-042],
-             [3.65130247e-001,
-              1.00000000e-300,
-              6.34869753e-001,
-              1.00000000e-300],
-             [8.68776164e-001,
-              1.02254304e-034,
-              1.31223836e-001,
-              6.21835051e-047],
-             [3.33333333e-301, 3.33333333e-001, 3.33333333e-301, 6.66666667e-001]])
+            [
+                [5.16919807e-001, 3.65825814e-033, 4.83080193e-001, 9.23220689e-042],
+                [3.65130247e-001, 1.00000000e-300, 6.34869753e-001, 1.00000000e-300],
+                [8.68776164e-001, 1.02254304e-034, 1.31223836e-001, 6.21835051e-047],
+                [3.33333333e-301, 3.33333333e-001, 3.33333333e-301, 6.66666667e-001],
+            ]
+        )
         output_p_emission = array(
-            [[2.02593570e-301, 2.02593570e-301, 2.02593570e-301, 1.00000000e+000],
-             [1.00000000e-300,
-              1.00000000e-300,
-              1.00000000e+000,
-              1.09629016e-259],
-             [3.26369779e-301,
-              3.26369779e-301,
-              3.26369779e-301,
-              1.00000000e+000],
-             [3.33333333e-001, 6.66666667e-001, 3.33333333e-301, 3.33333333e-301]])
+            [
+                [2.02593570e-301, 2.02593570e-301, 2.02593570e-301, 1.00000000e000],
+                [1.00000000e-300, 1.00000000e-300, 1.00000000e000, 1.09629016e-259],
+                [3.26369779e-301, 3.26369779e-301, 3.26369779e-301, 1.00000000e000],
+                [3.33333333e-001, 6.66666667e-001, 3.33333333e-301, 3.33333333e-301],
+            ]
+        )
 
         markov_model = MarkovModel.train_bw(states, alphabet, training_data)
         self.assertEqual("".join(markov_model.states), "".join(states))
         self.assertEqual("".join(markov_model.alphabet), "".join(alphabet))
-        self.assertTrue(array_equal(
-            around(markov_model.p_initial, decimals=3),
-            around(output_p_initial, decimals=3)))
-        self.assertTrue(array_equal(around(
-            markov_model.p_transition, decimals=3),
-            around(output_p_transition, decimals=3)))
-        self.assertTrue(array_equal(around(
-            markov_model.p_emission, decimals=3),
-            around(output_p_emission, decimals=3)))
+        self.assertTrue(
+            array_equal(
+                around(markov_model.p_initial, decimals=3),
+                around(output_p_initial, decimals=3),
+            )
+        )
+        self.assertTrue(
+            array_equal(
+                around(markov_model.p_transition, decimals=3),
+                around(output_p_transition, decimals=3),
+            )
+        )
+        self.assertTrue(
+            array_equal(
+                around(markov_model.p_emission, decimals=3),
+                around(output_p_emission, decimals=3),
+            )
+        )
```

Comment:
There is probably a numpy way to test two arrays are approximately equal... but that's a side issue.